features/step_definitions/environments.yml

```diff
@@ -3,21 +3,21 @@ local:
   test-rp: http://localhost:50130/test-rp
   idps:
     Stub Country: http://localhost:50140/eidas/stub-country/login
+    Stub Idp Demo: http://localhost:50140/stub-idp-demo/login
+    Post Office Stub: http://localhost:50140/post-office-test-rp/login
 
 docker-local:
   frontend: http://frontend
   test-rp: http://test-rp/test-rp
   idps:
     Stub Country: http://stub-idp/eidas/stub-country/login
+    Stub Idp Demo: http://stub-idp/stub-idp-demo/login
+    Post Office Stub: http://stub-idp/post-office-test-rp/login
 
 joint:
   frontend: https://www.joint.signin.service.gov.uk
   test-rp: https://test-rp-stub-joint.ida.digital.cabinet-office.gov.uk/test-rp
   idps:
     Stub Country: https://ida-stub-idp-joint.cloudapps.digital/eidas/stub-country/login
-
-staging:
```

Comment:
Removing this section has caused a problem.

This PR should not have been approved with this left in place. 

Please can we be more careful the next time.